views/scan.hbs

```diff
@@ -63,10 +63,11 @@
         <div class="flex">
           <h4>{{ name }}</h4><br>
           <p>
-            <span class="bold">Breach date:</span> {{ meta.date }}<br>
-            <span class="bold">Compromised data:</span> {{ meta.dataClasses }}<br>
-            <span class="bold">Compromised accounts:</span> {{ meta.acCount }}<br>
-            {{ meta.description }}
+            <span class="bold">Breach date:</span> {{prettyDate meta.BreachDate }}<br>
+            <span class="bold">Compromised data:</span> {{breachDataClasses meta.DataClasses }}<br>
+            <span class="bold">Compromised accounts:</span> {{localeString meta.PwnCount }}<br>
+            {{! TODO: make sure meta.Description is a safe string so we can use triple-{{{ }}
+            {{{ meta.Description }}}
```

Comment:
Should we use something like [**dompurify**](https://github.com/cure53/DOMPurify) to sanitize the `meta.Description`?

Reply:
We should. Should it be part of this PR or a new issue?

Reply:
I'll file a new issue, but we should probably address before release.